build.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+[[ -f /etc/prfile.d/modules.sh ]] && \
+  source /etc/profile.d/modules.sh
+
+builddir=build
+[ -d $builddir ] && rm -rf $builddir; mkdir $builddir
+
+if type module
+then
+  proj_dir=/qfs/projects/exasgd
+  mod_dir=/share/apps
+  cluster=$(uname -n | cut -d'.' -f1 | sed -E 's/[[:digit:]]//g')
+  module purge
+  if [ "$cluster" == "newell" ]
+  then
+    gcc=7.4.0
+    cmake=3.16.4
+    openmpi=3.1.5
+    cuda=10.2
+    magma=2.5.2_cuda10.2
+    metis=5.1.0
+  else
+    gcc=7.3.0
+    cmake=3.15.3
+    openmpi=3.1.3
+    cuda=10.1.243
+    magma=2.5.2_cuda10.2
+    metis=5.1.0
+  fi
```

Comment:
```suggestion
  else
    echo "Generic build ..."
  fi
```
The default option should not try to load any modules.

Reply:
Good call